### PR TITLE
Ab headlines test

### PR DESCRIPTION
**Pull request by gklopper**
Customisation to run at 50% of traffic on US network front

### CODE REVIEW

File: common/app/mvt/MultiVariateTesting.scala [hunk 1]
```diff
@@ -32,18 +32,26 @@ object JspmControlTest extends TestDefinition(
...
+  override def isParticipating(implicit request: RequestHeader): Boolean = {
+    request.headers.get("X-GU-hlt").contains("hlt-A") && switch.isSwitchedOn
+  }
+}
 
 object ABHeadlinesTestControl extends TestDefinition(
-  List(Variant6, Variant7, Variant8, Variant9, Variant0),
+  Nil,
   "headlines-ab-control",
   "To test how much of a difference changing a headline makes (test group)",
```
Comment (johnduffell) [inline common/app/mvt/MultiVariateTesting.scala]: control group surely, not test group? the variant is the test

File: common/app/mvt/MultiVariateTesting.scala [hunk 2]
```diff
@@ -32,18 +32,26 @@ object JspmControlTest extends TestDefinition(
...
   new LocalDate(2015, 9, 30)
-)
+) {
+  override def isParticipating(implicit request: RequestHeader): Boolean = {
+    request.headers.get("X-GU-hlt").contains("hlt-B") && switch.isSwitchedOn
```
Comment (johnduffell) [inline common/app/mvt/MultiVariateTesting.scala]: && conf.Switches.ServerSideTests.isSwitchedOn

**Comment by johnduffell** [comment]
:+1: looks like you have some random npm errors in your build too

File: common/app/mvt/MultiVariateTesting.scala (hunk 2 above)
Comment (gklopper) [inline common/app/mvt/MultiVariateTesting.scala]: Ah, good spot

**Pull request merged**